### ISSUE TITLE
How to reduce the config-connector-operator logging verbosity ?

### ISSUE THREAD

**Issue by fabito** [GoogleCloudPlatform/k8s-config-connector#475]
`config-connector-operator` is generating heaps of `DEBUG` level logging messages:

![2021-05-20_12-05](https://user-images.githubusercontent.com/308613/118899817-b7b62300-b963-11eb-889e-47c6d68ee626.png)

Is there a way to reduce the log level / verbosity  (environment variable or CLI argument) ?

**Comment by spew** [comment]
Hi @fabito thank you for bringing us this concern. We will consider either reducing the logging overall or adding controls to reduce the verbosity in the future.

To help: is this verbose logging creating any operational issues for you (i.e. an outage, etc) or is it more just making it hard to work with and view the logs?

**Comment by fabito** [comment]
Hi @spew ,

No, no operational issues related to config connector.
We were facing cost issues due to [this other issue](https://github.com/kubeflow/gcp-blueprints/issues/184) and while I was investigating it I noticed the high volume of debug messages generated by the operator.

**Comment by xiaobaitusi** [comment]
Hi @fabito , thanks for clarification. We will look into reducing the overall logging and will update this thread when we have more information.

**Comment by susan-shu-c** [comment]
Hey, same here, large amount of logs from

`profile-controller` and `metacontroller` (some sort of stderr logs)

**Comment by maqiuyujoyce** [comment]
Hi @susan-shu-c , thank you for your feedback! `profile-controller` and `metacontroller` seem to be outside of the scope of Config Connector, you might want to file issues to the corresponding owners. Or if you think it is Config Connector related, could you clarify a bit?

**Comment by lostick** [comment]
> To help: is this verbose logging creating any operational issues for you (i.e. an outage, etc) or is it more just making it hard to work with and view the logs?
@spew when thousands of CRDs are deployed, the volume of logs makes it difficult to capture anything. If we could set the  `severity` it would be really beneficial, thanks.

**Comment by xiaobaitusi** [comment]
@lostick , you are right, most of info logs are not that useful, we will consider reducing the logging overall as the first step to mitigate the issue. 

We might consider to adding the control of logging verbosity if there is strong need.

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.
